lib/hanami/cli/generators/app/slice/app_layout.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title><%= humanized_app_name %> - <%= humanized_slice_name %></title>
+    <%- if bundled_assets? -%>
+    <%%= favicon %>
+    <%= stylesheet_erb_tag %>
```

Comment:
Do we also want to generate a default tag to include the main app JS file?

Reply:
@timriley Yes, please see https://github.com/hanami/cli/pull/99/commits/ede966a514ba1b181c95ad7acccfa882dca5599e